### PR TITLE
edxapp migrations to run are now seperated in to file per database

### PR DESCRIPTION
**Pull request by macdiesel**
@edx/pipeline-team PTAL.

### CODE REVIEW

**Comment by doctoryes** [comment]
These changes make total sense and are correct. The first actual migration would have consumed the second unapplied migrations' output. 👍

**Comment by feanil** [comment]
:+1:

File: playbooks/continuous_delivery/run_migrations.yml [hunk 1]
```diff
@@ -79,8 +83,7 @@
       fail_on_missing: True
       mode: 0700
     with_items:
-      - "*{{ unapplied_migrations_output }}"
-      - "{{ migration_output }}"
+      - "{{ migration_files.stdout }}"
```
Comment (doctoryes) [inline playbooks/continuous_delivery/run_migrations.yml]: Perhaps this won't matter soon when migrations are run together, but it'd be nice to transfer the unapplied migrations to the artifacts dir _before_ running migrations. That way, even if migrations fail, the unapplied migrations will be available to view for debugging purposes.

Comment (macdiesel) [reply playbooks/continuous_delivery/run_migrations.yml]: Any task that fails within a job will cause the job to fail and artifact are not uploaded. Users will need to rely on the information in the console when a job fails. We should be sure to output any information needed to troubleshoot a defect to stderr/stdout.

Comment (doctoryes) [reply playbooks/continuous_delivery/run_migrations.yml]: 👍

**Pull request merged**